controllers/groupController.js

```diff
@@ -1,20 +1,26 @@
+import { cyan, red, green } from 'colors';
+import { ConsoleLogger } from '../utils/logger';
 import { createGroupService, getGroupsService, getGroupByIdService, updateGroupService, deleteGroupService } from '../services/groupService';
 
 export default class GroupController {
     async createGroup(req, res) {
         try {
-            const createdGroup = await createGroupService(req.body);
-            res.json(createdGroup);
+            const { execTime } = await createGroupService(req.body);
+            ConsoleLogger.info(`${cyan('Request')}: ${req.method} ${req.url}, ${cyan('Query')}: ${JSON.stringify(req.query)}, ${green('Execution time:')} ${execTime} ms`);

```

Comment:
It will be better if logger will be done as a middleware. Then all requests will be logged and if you will add new route it will be logged automatically and it will not be required to add log function manually.